src/data/actions/table.js

```diff
@@ -132,17 +99,13 @@ const sortData = (data, ordering) => {
   return orderBy(data, parseKeyValue, [direction]);
 };
 
-const sortTable = (tableId, fetchMethod, sortOptions) => (
+const sortTable = (tableId, fetchMethod, ordering) => (
   (dispatch, getState) => {
-    // Paragon Table's onSort passing in options: key, direction
-    // Our Api is expecting single orderField
-    const ordering = sortOptions.direction === 'desc' ? `-${sortOptions.key}` : sortOptions.key;
     const tableState = getState().table[tableId];
     const options = {
-      ...fetchOptions(tableState),
+      ...getPageOptionsFromQS(),
       ordering,
     };
-    updateUrl(options);
     dispatch(sortRequest(tableId, ordering));
```

Comment:
Unrelated to the changes in this PR, but do you know why we are passing `ordering` to the `sortRequest` action when the associated reducer ([`SORT_REQUEST`](https://github.com/edx/edx-portal/blob/master/src/data/reducers/table.js#L39)) does not set `ordering` in the Redux store?

Reply:
At one point we were setting the ordering in the `SORT_REQUEST`, put for `PAGINATION` actions we not setting it until we successfully got the data in a `PAGATION_REQUEST`.  It seems when I made the change to the reducer I missed the action handler. Will update

Reply:
We do this so that the events we publish to segment have the ordering in them.
<img width="344" alt="screen shot 2018-10-16 at 10 17 03 pm" src="https://user-images.githubusercontent.com/1643389/47058034-443b4c00-d191-11e8-83ea-193c2d607fb5.png">

Reply:
Oh, right. Thanks for the reminder! 😄 I wonder if we should add comments when we pass values to the actions that aren't used in the reducer as a reminder for why we're doing that, e.g., `// pass ordering for segment tracking`.